### ISSUE TITLE
Advanced merging and publishing of partitioned job results

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-aggregator#56]
from #52

> - metadata of final result: properly merge metadata of subjobs (e.g. STAC collection)
> - "publication" of results: e.g. VRT generation of (partial) result, or upload to a STAC catalog

**Comment by m-mohr** [comment]
Do I understand correctly that it is planned if a job is split and executed at EODC and VITO, the results stay in the data centers and the aggregator combines them just by providing a Collection pointing to the sub results? (If yes, great. If no, why?)

**Comment by soxofaan** [comment]
Indeed, merging at the metadata level and keeping the actual data as-is  would at least be the initial and obvious approach.

I remember people discussing in the forum (I think) about a feature to also stitch sub-results to a single geotiff or netcdf, but that's a lot less trivial to achieve, and no concrete plans there so far.
